climakitae/data_loaders.py

```diff
@@ -133,6 +135,49 @@ def _get_cat_subset(selections, cat):
         experiment_id=experiment_id,
         source_id=source_id,
     )
+
+    # Get just data that's on the LOCA grid
+    # This will include LOCA data and WRF data on the LOCA native grid
+    # Both datasets are tagged with UCSD as the institution_id, so we can use "UCSD" to further subset the catalog data
+    if "Statistical" in selections.downscaling_method:
+        cat_subset = cat_subset.search(institution_id="UCSD")
+    # If only dynamical is selected, we need to remove UCSD from the WRF query
+    else:
+        wrf_on_native_grid = [
+            institution
+            for institution in cat.df.institution_id.unique()
+            if institution != "UCSD"
+        ]
+        cat_subset = cat_subset.search(institution_id=wrf_on_native_grid)
+
+    # Limit simulations if both LOCA and WRF are selected
+    # LOCA has more simulations than WRF
+    # We just want the simulations that are present in both datasets
```

Comment:
I'm not sure about that. I think we'd want all of them returned. Hopefully the data structure will allow for that without too much inefficiency.

Reply:
> * I seem to recall that CESM2 doesn't have LOCA2 daily min and max temperature, but it _should_ have specific humidity, yet I don't see it among the simulations even when I select specific humidity

I noticed this too. I'll look into that.